cunit/syslog.c

```diff
@@ -63,8 +63,8 @@ extern int verbose;
 
 struct slmatch
 {
-    const char *re;         /* NULL => disabled */
-    int re_needs_free;      /* true if re is on the heap */
+    int is_used;            /* is this slot claimed? */
+    char re[1025];          /* NULL => disabled */
```

Comment:
I guess the comment needs to read "empty => disabled" now, since a char array can't be NULL.

Are we sure `re` is reliably initialized empty?  I don't see it happening in the diff, so probably the whole struct is memset or initalized to `{0}` somewhere which just didn't change, but good to check these things

Reply:
Yeah it's static, which means everything is initialized to 0